### ISSUE TITLE
Bug with creating an adjustment

### ISSUE THREAD

**Issue by BenGoldberg** [recurly/recurly-client-net#269]
I saw the below stack trace in the logs recently. This was using 1.9. We were previously using 1.8.1 and had no issues with it, so this appears to be a new bug related to the latest release.

```
System.ArgumentException: Cannot convert a null or empty string to an Enumeration.
Parameter name: source
 at Recurly.StringExtensions.ParseAsEnum[T](String source) in C:\\Users\\ben\\Documents\\GitHubVisualStudio\\recurly-client-net\\Library\\Extensions\\StringExtensions.cs:line 52
 at Recurly.Adjustment.ReadXml(XmlTextReader reader) in C:\\Users\\ben\\Documents\\GitHubVisualStudio\\recurly-client-net\\Library\\Adjustment.cs:line 239
 at Recurly.Client.ReadWebResponse(HttpWebResponse response, ReadXmlDelegate readXmlDelegate, ReadXmlListDelegate readXmlListDelegate, ReadResponseDelegate responseDelegate) in C:\\Users\\ben\\Documents\\GitHubVisualStudio\\recurly-client-net\\Library\\Client.cs:line 375
 at Recurly.Client.PerformRequest(HttpRequestMethod method, String urlPath, WriteXmlDelegate writeXmlDelegate, ReadXmlDelegate readXmlDelegate, ReadXmlListDelegate readXmlListDelegate, ReadResponseDelegate reseponseDelegate) in C:\\Users\\ben\\Documents\\GitHubVisualStudio\\recurly-client-net\\Library\\Client.cs:line 169
 at Recurly.Adjustment.Create() in C:\\Users\\ben\\Documents\\GitHubVisualStudio\\recurly-client-net\\Library\\Adjustment.cs:line 121
```

**Comment by BenGoldberg** [comment]
Is this fix included in the latest release of the NuGet build? If not, when will it be released?

**Comment by bhelx** [comment]
@BenGoldberg This fix is available in release [1.9.1](https://github.com/recurly/recurly-client-net/releases/tag/1.9.1)
